.github/dependabot.yml

```diff
@@ -0,0 +1,24 @@
+version: 2
+updates:
+- package-ecosystem: "gomod"
+  directory: "/"
+  schedule:
+    interval: "weekly"
+
+- package-ecosystem: "docker"
+  directory: "/"
+  schedule:
+    interval: "weekly"
+
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    interval: "weekly"
+  groups:
+    k8s:
+      patterns:
+      - "k8s.io/*"
+      - "sigs.k8s.io/*"
+    onsi:
+      patterns:
+      - "github.com/onsi/*"
```

Comment:
I am a little confused. Usually `github-actions` only focuses on `.github/workflows`; but `"k8s.io/*"`, `"sigs.k8s.io/*"` and `"github.com/onsi/*"` are dependencies used in golang source code. Did I miss anything?

Reply:
Good catch, moved to the `gomod` section.